packages/teams-js/test/utils.ts

```diff
@@ -2,20 +2,34 @@ import { validOriginsFallback as validOrigins } from '../src/internal/constants'
 import { defaultSDKVersionForCompatCheck } from '../src/internal/constants';
 import { GlobalVars } from '../src/internal/globalVars';
 import { DOMMessageEvent, ExtendedWindow } from '../src/internal/interfaces';
-import { MessageResponse } from '../src/internal/messageObjects';
+import { MessageID, MessageResponse, MessageUUID, SerializedMessageRequest } from '../src/internal/messageObjects';
 import { NestedAppAuthRequest } from '../src/internal/nestedAppAuthUtils';
 import { HostClientType } from '../src/public';
 import { app } from '../src/public/app';
 import { applyRuntimeConfig, IBaseRuntime, setUnitializedRuntime } from '../src/public/runtime';
 
 export interface MessageRequest {
```

Comment:
>MessageRequest [](http://example.com/codeflow?start=17&length=14)

How is this interface different than the one `messageObjects.ts`? Should this one just be importing and using that one, instead of duplicating it?

Reply:
I agree. Let me remove this.

Reply:
Looks like MessageRequest doesn't have the property isPartialResponse which is used in the PrivateAPI spec. I will rework those tests to make sure that they are done correctly. Since `isPartialResponse` should be only for `MessageResponses`